Rakefile

```diff
@@ -26,8 +26,22 @@ task :generate do
         imports: ['java.time.LocalDateTime'],
       )
     ]
-  )
-  generator.save('support/src/test/java/com/shopify/graphql/support/Generated.java')
+  ).save('support/src/test/java/com/shopify/graphql/support/GeneratedExampleSchema.java')
+
+  #Schema without mutations
+  GraphQLJavaGen.new(GraphQLSchema.new(Support::Schema.introspection_result(Support::Schema::NoMutationSchema)),
+    package_name: 'com.shopify.graphql.support',
+    nest_under: 'GeneratedNoMutationSchema',
+    custom_scalars: [
```

Comment:
The custom scalars are already tested in the other generated schema.  They could be omitted from this schema.